README.md

```diff
@@ -138,7 +138,7 @@ http://xeus-cpp.readthedocs.io
 
 | `xeus-cpp` | `xeus-zmq`      | `CppInterOp` | `pugixml` | `cpp-argparse`| `nlohmann_json` |
 |------------|-----------------|--------------|-----------|---------------|-----------------|
-|  main      |  >=3.0.0,<4.0.0 | >=1.5.0      | ~1.8.1    | >=3.0,<4.0    | >=3.11.3,<4.0   |
+|  main      |  >=3.0.0,<4.0.0 | >=1.7.0      | ~1.8.1    | >=3.0,<4.0    | >=3.12.0,<4.0   |
```

Comment:
The latest xeus release forces a requirement on nlohmann_json 3.12.0 (taking a chance to address this in the readme too)